### ISSUE TITLE
Missing string from translation file

### ISSUE THREAD

**Issue by modelm** [r-a-y/bp-reply-by-email#108]
I would like to translate the following string found in https://github.com/r-a-y/bp-reply-by-email/blob/master/includes/bp-rbe-functions.php#L317:

`__( '--- Replying to this email will not send a message directly to the recipient or group ---', 'bp-rbe' )`

It's not in https://github.com/r-a-y/bp-reply-by-email/blob/master/languages/bp-rbe.pot.

**Comment by r-a-y** [comment]
Thanks for opening.  I know I don't update the POT file as often as I should.

I'll probably update it once I've added SparkPost support to attachments (#104).

**Comment by modelm** [comment]
Sounds good, I went ahead and generated a temporary current POT file from the latest master in case anybody needs it: https://gist.github.com/modelm/b2409848b8ba082f00005ceaf8a83f34

**Comment by modelm** [comment]
Using my translation file to translate this particular string, I found there is a character limit that breaks replying because of a line break preventing the string matching (I think). When it's long enough to break a line, I get a "Your Reply By Email message could not be posted" error back and this is the quoted message:

```
Your reply could not be posted because we could not find the "--- Reply to this e-mail to send your response to the Commons (attachments coming soon!) ---" marker in the body of your email.
...
On Mon, Dec 18, 2017 at 1:55 PM, Ryan Williams 
wrote:

> --- Reply to this e-mail to send your response to the Commons (attachments
> coming soon!) ---
>
```

I'm not sure if it this problem affects only certain email clients or what actually causes the line break. If I shorten the translation so that it doesn't break a line, replying works fine.

**Comment by r-a-y** [comment]
The main issue is fixed in #112.

The other issue mentioned in the last comment is tracked in #109.
